load/load_final_table/lambda_function.py

```
@@ -267,10 +267,12 @@ def load_redshift_table(df_view_table, redshift_conn, event):
         batch_time = event["batch_period"].split("_")[1]
         batch_time_dt = datetime.datetime.strptime(batch_time, "%Y-%m-%d-%H-%M-%S")
         top_comm = row["top_comm"]
+        comm_positive_count = row['comm_positive_count']
```

Ruff (0.8.2)

Local variable `comm_positive_count` is assigned to but never used

Remove assignment to unused variable `comm_positive_count`

(F841)